### PR TITLE
FLock template

### PR DESCRIPTION
**Pull request by fengzz-coding**
FLock template add.

### CODE REVIEW

File: FLock/README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-# Usage
+# ⚡Usage
 
-FLOCK_API_KEY And task_id get from [FLock](https://www.flock.io),And fill in the deploy.yaml. 
\ No newline at end of file
+FLOCK_API_KEY And task_id get from [FLock](https://beta.flock.io/),and fill in the deploy.yaml.
+
+# 📖 Document
+
+[FLock Docs](https://docs.flock.io/ai-co-creation-platform/guidelines)
```
Comment (mattflock) [inline FLock/README.md:11]: Update link to https://docs.flock.io/flock-product/decentralised-ai-training/validator-guide

The current one is to the co-creation platform which might be a little confusing for validators

File: FLock/README.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
-# FLock llm-loss-validator
+# 🔒FLock llm-loss-validator
 
 Validator that computes the validation loss for a huggingface-compatible LLM
 
-# Usage
+# ⚡Usage
 
-FLOCK_API_KEY And task_id get from [FLock](https://www.flock.io),And fill in the deploy.yaml.
```
Comment (mattflock) [inline FLock/README.md]: update link to https://train.flock.io

**Comment by anilmurty** [comment]
Thanks for this @fengzz-coding - a couple comments (as discussed in slack):
1. Will be great if you could add a little more context (at least what and why. Ideally a video if possible) in the readme. This will help boost usage of it. Here is a good recent example of one https://github.com/akash-network/awesome-akash/blob/master/vllm/README.md 
2. May want to consider not filtering GPUs - so user gets more bids

**Comment by anilmurty** [comment]
@Dimokus88 - could you please review/ test/ merge when you have the chance?

**Comment by Dimokus88** [comment]
The main check is in progress.
There are currently some issues that need to be addressed before inclusion in the awesome-akash repository:

1. Add a Dockerfile of the image public.ecr.aws/e7z6j8c3/flock:llm-loss-validator-gpu-latest . Also, if there is an executable script inside the image, they also need to be added as a file.

2. You need to add information about your template to the root README.md of the awesome-akash repository

**Comment by Dimokus88** [comment]
> 1. For the Docker build, you can refer to this https://github.com/FLock-io/llm-loss-validator/blob/main/Dockerfile-gpu.  What else needs to be provided for this?

It is necessary that users know what kind of image will be deployed (and, if desired, could assemble it themselves). This is why there is a need to publish a Dockerfile for custom images. Or add a link to the Dockerfile in your README.md

**Comment by Dimokus88** [comment]
SDL looks working.
But it is necessary to clarify/correct the recommendations above.
Adding information to `README.md` and publishing a `Dockerfile` (or a link to it). Users need to see what they are deploying.

**Comment by Dimokus88** [comment]
Thx @fengzz-coding !
Looks ready to merge @anilmurty !

**Comment by anilmurty** [comment]
Thanks @Dimokus88

**Review by anilmurty — APPROVED** [review]

**Pull request merged**